Zapper-fi/studio
# feat(dolomite): Added Dolomite to Zapper ⛰️

## Description



## Checklist

- [x] (optional) As a contributor, my Ethereum address/ENS is: dolomite-exchange.eth
- [x] (optional) As a contributor, my Twitter handle is: @Dolomite_io

## How to test?

Use the following address `0x52256ef863a713Ef349ae6E97A7E8f35785145dE` to check returned data from the API.

You can double check the values are correct by impersonating the account via [Impersonator](https://www.impersonator.xyz/) on app.dolomite.io.
